### ISSUE TITLE
I'd like to make a Korean version of the Electronic Observer

### ISSUE THREAD

**Issue by MirhasVida** [ElectronicObserverEN/ElectronicObserver#467]
I can help you with English to Korean and Japanese to Korean translation.
If you are interested in my proposal, please get in touch with me.

**Comment by myangelkamikaze** [comment]
Our translations are split in 2 repositories, actually.

Most game related data is in our [data repository](https://github.com/ElectronicObserverEN/Data). For Korean the files are already set up, so you'd just need to make a PR (if you're unfamiliar with how git works, you can just send us the updated files either here or on discord).

For Electronic Observer itself, we use [resx manager](https://github.com/dotnet/ResXResourceManager) which is available as a standalone application or a Visual Studio extension if you're using it.

**Comment by MirhasVida** [comment]
So you meant that all I have to do is edit the data repository files in Korean, right?

**Comment by myangelkamikaze** [comment]
Yeah, you'd just need to update all the [files](https://github.com/ElectronicObserverEN/Data/tree/master/Translations/ko-KR) to have Korean translations instead of English.
